### ISSUE TITLE
CosmosDB x-ms-retry-after-ms header access

### ISSUE THREAD

**Issue by sergsalo** [jbmusso/gremlin-javascript#113]
CosmosDB can reject the request with 429 status code. Here is the sample response:

ExceptionType : RequestRateTooLargeException
ExceptionMessage : Message: {"Errors":["Request rate is large"]}

According to documentation: 
"When a client attempts to exceed the reserved throughput for an account, there is no performance degradation at the server and no use of throughput capacity beyond the reserved level. The server will preemptively end the request with RequestRateTooLarge (HTTP status code 429) and return the x-ms-retry-after-ms header indicating the amount of time, in milliseconds, that the user must wait before reattempting the request."

Is there a way to access header "x-ms-retry-after-ms" in gremlin execute response to handle the retry properly ?

so, basically I'm running 
`graph.execute("g.E().count()", {}, (err, results) =>{...})`

  - how to get that header from here?

Thanks,

**Comment by AbdolrezaMadi** [comment]
I'm looking for the same thing. Is there any update on this issue?

**Comment by NoamWe** [comment]
Would be happy to hear an update regarding this issue too.
